package.json

```diff
@@ -26,7 +26,21 @@
       "default": "./lib/universal.js"
     }
   },
+  "typesVersions": {
+    "*": {
+      "use-at-your-own-risk": [
+        "./lib/types/use-at-your-own-risk.d.ts"
+      ],
+      "rules": [
+        "./lib/types/rules/index.d.ts"
+      ],
+      "universal": [
+        "./lib/types/universal.d.ts"
+      ]
+    }
+  },
   "scripts": {
+    "are-the-types-wrong": "attw --pack",
```

Comment:
By [our conventions](https://eslint.org/docs/latest/contribute/package-json-conventions#main-script-names), the name of this script should start with `lint:` or `test:`. I'd tend towards `lint:` because it's not actually running any module code (this can be trivially verified by adding a `throw` statement at the top of lib/unsupported-api.js for example), but only analyzing file contents.

Reply:
How about `check:types`?

Reply:
`lint:types` would fit our conventions.

Reply:
Sure thing.